ql/src/test/results/clientpositive/llap/udf5.q.out

```diff
@@ -260,7 +260,7 @@ POSTHOOK: query: select from_unixtime(unix_timestamp('1400-11-08 08:00:00 ICT',
 POSTHOOK: type: QUERY
 POSTHOOK: Input: _dummy_database@_dummy_table
 #### A masked pattern was here ####
-1400-11-08 08:00:00
+1400-11-08 07:35:24
```

Comment:
This value doesn't seem to be honoring +06:42:04. Why is that?

Reply:
Because input is ICT(+07:00) output is Asia/bangkok(+6:42:04) due to that we are seeing this difference.